[2022-10-28T21:36:36.859+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: carga_tb_carteira.create_datalake manual__2022-10-28T21:36:33.463597+00:00 [queued]>
[2022-10-28T21:36:36.960+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: carga_tb_carteira.create_datalake manual__2022-10-28T21:36:33.463597+00:00 [queued]>
[2022-10-28T21:36:36.960+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-28T21:36:36.960+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-10-28T21:36:36.960+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-28T21:36:36.977+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): create_datalake> on 2022-10-28 21:36:33.463597+00:00
[2022-10-28T21:36:36.982+0000] {standard_task_runner.py:55} INFO - Started process 4366 to run task
[2022-10-28T21:36:36.986+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'carga_tb_carteira', 'create_datalake', 'manual__2022-10-28T21:36:33.463597+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/carga_tb_carteira.py', '--cfg-path', '/tmp/tmp1skhw05l']
[2022-10-28T21:36:36.989+0000] {standard_task_runner.py:83} INFO - Job 28: Subtask create_datalake
[2022-10-28T21:36:37.047+0000] {task_command.py:376} INFO - Running <TaskInstance: carga_tb_carteira.create_datalake manual__2022-10-28T21:36:33.463597+00:00 [running]> on host cs-1097450612417-default
[2022-10-28T21:36:37.111+0000] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=Marcelo Henrique
AIRFLOW_CTX_DAG_ID=carga_tb_carteira
AIRFLOW_CTX_TASK_ID=create_datalake
AIRFLOW_CTX_EXECUTION_DATE=2022-10-28T21:36:33.463597+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-28T21:36:33.463597+00:00
[2022-10-28T21:36:37.113+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-10-28T21:36:37.121+0000] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=carga_tb_carteira, task_id=create_datalake, execution_date=20221028T213633, start_date=20221028T213636, end_date=20221028T213637
[2022-10-28T21:36:37.158+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-10-28T21:36:37.188+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
